packages/auth/src/core/auth/auth_impl.ts

```diff
@@ -167,7 +171,11 @@ export class AuthImpl implements AuthInternal, _FirebaseService {
         }
       }
 
-      await this.initializeCurrentUser(popupRedirectResolver);
+      // Skip loading users from persistence in FirebaseServerApp Auth instances.
```

Comment:
Could you elaborate why? What would happen when user signs in with SSR for the first time, then close and reopen the app?

Reply:
For this feature to work, the user has to be signed in on the client first and then an Auth ID Token is provided to the FirebaseServerApp. The design principal is for the FirebaseServerApp user to be directly soley by this AuthIdToken, and not be affected by persistence or any other Auth operations that change the user.

What we don't want is for persisted data to affect the current user  - ie a race condition where an AuthIDToken is defined and being loaded asynchronously while another user object is loaded from persistence. Only one should be applicable, and due to the aforementioned design decision, the applicable one is always the Auth ID Token.